packages/make-styles/src/runtime/expandShorthand.ts

```diff
@@ -16,15 +16,19 @@ export function expandShorthand(style: MakeStylesStyle, result: MakeStylesStyle
       if (expansion) {
         Object.assign(result, expansion);
       } else {
-        result[property as keyof MakeStylesStyle] = value;
+        // Cast to any to avoid "error TS2590: Expression produces a union type that is too complex to represent"
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (result as any)[property as keyof MakeStylesStyle] = value;
```

Comment:
Would also like to find a better type here

Reply:
This file will be removed entirely in #20539, so probably no need 😁